### ISSUE TITLE
question about cfg pretrained model

### ISSUE THREAD

**Issue by lixingyi8947** [facebookresearch/maskrcnn-benchmark#415]
## ❓ Questions and Help
I saw there are two pretrained version of MaskRCNN in both configs/e2e_mask_rcnn_R_101_FPN_1x.yaml
and configs/caffe2/e2e_mask_rcnn_R_101_FPN_1x_caffe2.yaml
What is the main difference between cfgs in configs and configs/caffe2?

**Comment by fmassa** [comment]
Hi,

The difference between both is that `configs/e2e_mask_rcnn_R_101_FPN_1x.yaml` is not trained on detection yet, while `configs/caffe2/e2e_mask_rcnn_R_101_FPN_1x_caffe2.yaml` was trained on detection using the pre-trained weights from Detectron.

If you want to have the pre-trained weights from the model trained in PyTorch, just download the weights from https://github.com/facebookresearch/maskrcnn-benchmark/blob/master/MODEL_ZOO.md#end-to-end-faster-and-mask-r-cnn-baselines and modify the config from `configs/` to point to the downloaded weight.

**Comment by lixingyi8947** [comment]
Thanks a lot! May I ask that is the model trained on Pytorch has same performance as trained on Caffe?
 I guess caffe model is better.
> Hi,
> 
> The difference between both is that `configs/e2e_mask_rcnn_R_101_FPN_1x.yaml` is not trained on detection yet, while `configs/caffe2/e2e_mask_rcnn_R_101_FPN_1x_caffe2.yaml` was trained on detection using the pre-trained weights from Detectron.
> 
> If you want to have the pre-trained weights from the model trained in PyTorch, just download the weights from https://github.com/facebookresearch/maskrcnn-benchmark/blob/master/MODEL_ZOO.md#end-to-end-faster-and-mask-r-cnn-baselines and modify the config from `configs/` to point to the downloaded weight.

**Comment by fmassa** [comment]
Hi @lixingyi8947 

As you can see from the [MODEL_ZOO](https://github.com/facebookresearch/maskrcnn-benchmark/blob/master/MODEL_ZOO.md#accuracy), the models trained with `maskrcnn-benchmark` match the accuracies of those trained with Detectron in Caffe2.
